api/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:7.4.27-apache-bullseye as base
```

Comment:
Chose to use the most explicit container tag so versions don't float

Reply:
Yeah, good idea.